frontend/test/pytest/test_measurement_primitives.py

```diff
@@ -35,9 +37,169 @@ def f():
         return sample_p.bind(obs, shots=5, shape=(5, 0))
 
     jaxpr = jax.make_jaxpr(f)().jaxpr
-    assert jaxpr.eqns[1].primitive == sample_p
-    assert jaxpr.eqns[1].params == {"shape": (5, 0), "shots": 5}
-    assert jaxpr.eqns[1].outvars[0].aval.shape == (5, 0)
+    # breakpoint()
+    mlir = lower_jaxpr_to_mlir(jax.make_jaxpr(f)(), "foo")[0]
+    # breakpoint()
+    assert (
+        jaxpr
+        == """
+{ lambda ; . let
+    a:AbstractObs(num_qubits=0,primitive=compbasis) = compbasis
+    b:f64[5,0] = sample[shape=(5, 0) shots=5] a
```

Comment:
The shots here for concrete shot values should also be changed to arg instead of kwarg when binding